mkosi/config.py

```diff
@@ -498,6 +499,33 @@ def match_systemd_version(value: str) -> bool:
     return config_match_version(value, version)
 
 
+def config_parse_bytes(value: Optional[str], old: Optional[int] = None) -> Optional[int]:
+    if not value:
+        return None
+
+    if value.endswith("G"):
+        factor = 1024 ** 3
+    elif value.endswith("M"):
+        factor = 1024 ** 2
+    elif value.endswith("K"):
+        factor = 1024
+    else:
+        factor = 1
+
+    if factor > 1:
+        value = value[:-1]
+
+    result = math.ceil(float(value) * factor)
+    if result <= 0:
+        die("Size out of range")
+
+    rem = result % 512
```

Comment:
I vaguely remember a discussion recently to keep partitions aligned to 4K, so should we also round here to the next 4K block?